pmm/pmm2-ami.groovy

```diff
@@ -46,67 +35,42 @@ pipeline {
                 expression { env.RELEASE_CANDIDATE == "yes" }
             }
             steps {
-                withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'pmm-staging-slave', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
-                    sh """
-                        set -o pipefail
-                        ~/bin/packer build \
-                        -var 'pmm_client_repos=original testing' \
-                        -var 'pmm_client_repo_name=percona-testing-x86_64' \
-                        -var 'pmm2_server_repo=testing' \
-                        -only amazon-ebs -color=false packer/pmm2.json \
-                            | tee build.log
-                    """
+                sh 'make pmm2-ami-rc'
```

Comment:
this approach could be cleaner, but for me, it decreases visibility to what repo values we are using for what parameter.

Reply:
I changed this because now we can run the build locally, just like we do in the pipeline. I think it's useful for debugging. But I agree that it slightly reduces visibility :(